src/commands/api-mesh/create.js

```diff
@@ -157,10 +157,18 @@ class CreateCommand extends Command {
 									? MULTITENANT_GRAPHQL_SERVER_BASE_URL
 									: meshURL;
 
-							this.log(
-								'Mesh Endpoint: %s\n',
-								`${meshUrl}/${mesh.meshId}/graphql?api_key=${adobeIdIntegrationsForWorkspace.apiKey}`,
-							);
+							if (
+								meshUrl === TMOConstants.TMO_STAGE_URL ||
+								meshUrl === TMOConstants.TMO_SANDBOX_URL ||
+								meshUrl === TMOConstants.TMO_PROD_URL
```

Comment:
This check could grow somewhat unruly if we add more TI environments, but it works for the time being since we only have TMO.